packages/server-wallet/src/protocols/__test__/application.test.ts

```diff
@@ -1,17 +1,15 @@
-import {simpleEthAllocation, makeDestination, BN, Uint256} from '@statechannels/wallet-core';
+import {simpleEthAllocation, BN, Uint256} from '@statechannels/wallet-core';
 import matchers from '@pacote/jest-either';
 import {some, none} from 'fp-ts/lib/Option';
 
 import {protocol} from '../application';
-import {alice} from '../../wallet/__test__/fixtures/signingWallets';
+import {alice} from '../../wallet/__test__/fixtures/participants';
```

Comment:
Maybe we should create some kind of `shared` folder for re-used test fixtures across unit tests, seems odd for one test file, nested under `protocols` to reach for fixtures from another context designed for `wallet`.

Reply:
My reasoning at the time is that fixtures should be defined next to the data structure they return. That approach seems reasonable, and so does a centralized approach that defines all the fixtures in one place.

Reply:
I think my point then becomes that we should have a shared folder for data types re-used in multiple contexts

Reply:
For example 
```
.
├── protocols
├── shared
│   ├── __test__
│   │   └── fixtures
│   └── types.ts
├── store
│   └── models
└── wallet

```